src/Footer/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './style.scss';
+import {
+  image5, icons,
+}
+  from '../variables/export-img';
+
+const Footer = () => (
```

Comment:
export here. use one convention

Reply:
i have used one convention,when we export afrer component